stickyuploads/storage.py

```diff
@@ -12,4 +15,18 @@ def __init__(self):
 
     def url(self, name):
         """Files are not accessable via url."""
-        raise NotImplementedError()
\ No newline at end of file
+        raise NotImplementedError()
+
+    def get_available_name(self, name, max_length=None):
+        """Return path to name placed in two random directories"""
+        name = os.path.join(
```

Comment:
I agree it would be helpful to have this project clean up its temp files more easily. I don't think it's a good idea to try to create our own version of `mkdtemp`.